src/lib/kit/hooks/useOneOf/useOneOf.scss

```diff
@@ -8,5 +8,22 @@
                 max-width: unset;
             }
         }
+
+        &_card {
+            & + .#{$ns}group-indent {
+                & > .#{$ns}use-search:not(.#{$ns}group-indent) {
+                    padding-top: 0px;
+                    margin-top: 15px;
+                }
```

Comment:
I can't imagine what it's for, can you show it?

Reply:
with this style
<img width="761" alt="image" src="https://github.com/gravity-ui/dynamic-forms/assets/23384737/460da03d-97aa-43e1-bbcd-36f1b52b61b3">
without this style
<img width="761" alt="image" src="https://github.com/gravity-ui/dynamic-forms/assets/23384737/5307c9e6-8a42-41d1-b60e-991c31dfe063">